lib/generators/aeternitas/templates/initializer.rb

```diff
@@ -1,4 +1,11 @@
 # Configure Aeternitas
 Aeternitas.configure do |config|
-  config.redis = {host: "localhost", port: 6379} # this is the default Redis config which should work in most cases.
+  # Configure source storage
+  # config.storage_adapter_config = {
+  #   directory: File.join(Rails.root, 'public', 'sources')
+  # }
```

Comment:
Just to check: Is this the default? Or does this have to be set.

It would be nice if a user, after installing this gem can immediately use it with sensible defaults (convention over configuration :).

The default for the directory should be the `storage` directory (rails default since a few major versions) with an `aeternitas` subdirectory in it.